docs/release-notes/cloud/cloud-2023-10-13-bundler-audit-deprecation.md

```diff
@@ -7,12 +7,14 @@ rss_href: /feed_rss_created.xml
 
 On October 13th 2023 we deprecated the tool **bundler-audit** in favor of [**Trivy**](https://github.com/codacy/codacy-trivy), a more complete and actively maintained tool for detecting vulnerabilities in Ruby gems and other languages, with a vulnerability database that's updated daily.
 
-## Retirement of bundler-audit January 1, 2024
+## Removal of bundler-audit January 1, 2024
 
-The retirement of the tool **bundler-audit** is scheduled for January 1st, 2024.
+On January 1st, 2024 we'll be removing **bundler-audit** from Codacy.
```

Comment:
When reading the original sentence, my first impression was that **bundler-audit** would be retired by the author, and I was trying to find that information so we could add a link to the source.

Now I understand that we are retiring this tool from Codacy, right? 🤔 
With that in mind, I'm proposing this change for consistency with [this one](https://docs.codacy.com/release-notes/cloud/cloud-2020-03-09-nodesecurity-golint-scsslint-removal/).

Does this make sense, @nicklem ?